crypto/src/test/java/com/virgilsecurity/sdk/crypto/VirgilCryptoTest.java

```diff
@@ -157,10 +172,9 @@ public void decrypt_verbose_error(VirgilCrypto crypto) throws VirgilException {
 
     VirgilKeyPair keyPairWrong = crypto.generateKeyPair();
 
-    assertEquals(assertThrows(DecryptionException.class, () -> {
+    String exceptionMessage = assertThrows(DecryptionException.class, () -> {
```

Comment:
exceptionMessage is not verified

Reply:
we don't need this test any more at all. We don't provide custom message anymore, we're just using the one that comes from the low-level crypto